tutorials/v7/ntuple/ntpl008_import.C

```diff
@@ -47,10 +47,10 @@ void ntpl008_import()
    // RNTupleImporter::Create() returns a wrapper for an RNTupleImporter. If the creation fails,
    // the result will contain an exception. Calling Unwrap() returns the actual RNTupleImporter
    // class or throws the exception in case of errors.
-   auto importer = RNTupleImporter::Create(kTreeFileName, kTreeName, kNTupleFileName).Unwrap();
+   auto importer = RNTupleImporter::Create(kTreeFileName, kTreeName, kNTupleFileName);
 
    // Using `ThrowOnError()` makes sure that the macro abort if the import fails.
-   importer->Import().ThrowOnError();
+   importer->Import();
```

Comment:
We'll need a fix-up of the code comments, too.

Reply:
Whoops, completely overlooked those. I also thought it might be nice to demonstrate the use of `RNTupleWriteOptions` with the importer so I added some lines, let me know if you think it's useful.